build-manual

```diff
@@ -11,6 +11,6 @@ ESCAPED_REQS=$(awk '{printf "%s%s",sep,$0; sep="\\\n"} END{print ""}' ../manual_
 sed "s/REQUIREMENTS/$ESCAPED_REQS/" index-rst-template > index.rst
 
 rm -rf build/
-sphinx-build -b html . build -W
```

Comment:
There are some warnings mentioned in https://github.com/CQCL/pytket-docs/issues/285

Disabling this for now

Reply:
actually to be more specific the warning is 

```
WARNING: Cell printed to stderr:
/Users/callum/Desktop/pytket-docs/.venv/lib/python3.11/site-packages/cotengra/hyperoptimizers/hyper.py:34: UserWarning: Couldn't import `kahypar` - skipping from default hyper optimizer and using basic `labels` method instead.
  warnings.warn(
```

This is a known issue with `pytket.zx` and ddon't think we can do anything about it for now.

Reply:
This issue was fixed in https://github.com/CQCL/pytket-docs/pull/315 by installing the Kayphar package in the manual build. 

We can add the -W flag now. the build is now tested properly on CI.